sunpy/map/tests/test_mapbase.py

```diff
@@ -1603,3 +1603,11 @@ def test_only_cd():
     cd_map = sunpy.map.Map((data, header))
     np.testing.assert_allclose(u.Quantity(cd_map.scale).value, np.array([5, 13]))
     np.testing.assert_allclose(cd_map.rotation_matrix, np.array([[3/5, -4/5], [5/13, 12/13]]))
+
+
+def test_basecoordianteframe_submap(aia171_test_map):
```

Comment:
Is there a better place to for this test to other than at the end of this file?

Reply:
I tried to find similar tests and put them besides them. There are some tests that have the same parameter,but they don't look like similar tests

Reply:
I added the modifications @ayshih